src/services/authentication.service.js

```diff
@@ -106,17 +113,35 @@ export default class UserService {
     return updatedUser;
   }
 
-  static updateIsVerified = async (email) => {
-    const update = await user.update(
-      { isVerified: true },
-      { where: { email } }
-    );
+  /**
+   * @param {string} value
+   * @param {boolean} status
+   * @returns {object} returns an updated user
+   */
+  static updateIsVerifiedOrDisableNotification = async (value, status) => {
```

Comment:
Could you change the name of this method and update it's JSDoc to reflect what it does exactly?